docs/design/autoware-interfaces/ad-api/features/driving.md

```diff
@@ -0,0 +1,31 @@
+# Driving API
+
+## Description
+
+The driving feature manages whether the vehicle is driving or can start driving.
+The vehicle holds a stop when the state is not DRIVING.
+This state is mainly used to show the users whether the vehicle can start driving and to implement an HMI for departure.
+
+## Driving State
+
+The state transition for driving state is illustrated in the diagram below.
+In a normal operation, this state will transition as follows:
+
+1. Driving state is initialized as NOT_READY
+2. State transitions to READY when Autoware components are launched successfully and is ready to start driving.
+3. The state becomes DRIVING if the engage API is called and the vehicle starts driving.
+4. When the vehicle reaches to the destination, the state returns to NOT_READY.
+5. The state can be manually returned to NOT_READY whenever disengage API is called
+
+![driving-state](./driving-state.drawio.svg)
+
+| State     | Description                                       |
+| --------- | ------------------------------------------------- |
+| NOT_READY | The vehicle cannot start driving for some reason. |
```

Comment:
Assuming NOT_READY is the opposite of READY, then the description should also be the opposite

```suggestion
| NOT_READY | The vehicle is not ready to start driving. |
```